### PR TITLE
gh-1580 rocks reference update

### PR DESCRIPTION
**Pull request by Onvember**
[x] Add reference for luatest and cartridge (just a link)

### CODE REVIEW

File: doc/reference/reference_rock/index.rst [hunk 1]
```diff
@@ -10,6 +10,6 @@ This reference covers third-party Lua modules for Tarantool.
     dbms
     expirationd
     membership
-    shard
     vshard/index
-    tdb
+    ../../book/cartridge/index
```
Comment (rosik) [inline doc/reference/reference_rock/index.rst:14]: Is it intentional?

**Review by rosik — COMMENTED** [review]
In general it seems OK. Is there any simple way to preview compiled doc?

File: update_submodules.sh [hunk 2]
```diff
@@ -31,12 +39,8 @@ mkdir -p "${cartridge_cli_dest}"
 yes | cp -rf "${cartridge_cli_root}/README.rst" "${cartridge_cli_index_dest}"
 
 cd "${cartridge_root}" || exit
-tarantoolctl rocks install \
-  https://raw.githubusercontent.com/tarantool/LDoc/tarantool/ldoc-scm-2.rockspec \
-  --server=http://rocks.moonscript.org
```
Comment (rosik) [inline update_submodules.sh:36]: Where's it installed now?

Comment (Onvember) [reply update_submodules.sh:36]: in Dockerfile

File: doc/reference/reference_rock/index.rst (hunk 1 above)
Comment (Onvember) [reply doc/reference/reference_rock/index.rst:14]: yes, @artur-barsegyan asked me to remove tbd and add cartridge here

**Pull request merged**